Tasks/ServiceFabricUpdateAppVersions/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -32,6 +32,7 @@
   "loc.messages.PreviousBuildNumberLabel": "Previous build number: '{0}'",
   "loc.messages.PreviousBuildLocationLabel": "Previous build location: '{0}'",
   "loc.messages.NoPreviousSuccessfulBuild": "Could not find previous build to compare against.",
+  "loc.messages.NoManifestInPreviousBuild": "Previous build does not contains application manifest. Force updating application type version.",
```

Comment:
Your message mentions only forcing update of the application manifest, but the code forces update of all manifests.